kong/api/endpoints.lua

```diff
@@ -304,22 +304,28 @@ end
 local function get_collection_endpoint(schema, foreign_schema, foreign_field_name, method)
   return not foreign_schema and function(self, db, helpers)
     local next_page_tags = ""
+    local next_page_size = ""
 
     local args = self.args.uri
     if args.tags then
       next_page_tags = "&tags=" .. escape_uri(type(args.tags) == "table" and args.tags[1] or args.tags)
     end
 
+    if type(args.size) == "number" then
```

Comment:
@fffonion I changed this validation in one of the last commits... I believe I need to update this Pull Request

Reply:
@jacksjm I tried to rebase but maybe i lost some of your commits. Could you rebase your PR upon master (you can use this PR as start and force push to yours)? Make sure not to use "merge" but "rebase", since we only merge PR with a rebase, if the PR contains merge commits, it may lead to conflicts as order changes.

Reply:
nvm, looks like i can do a squash : ) closing this now